<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolsClass</class>
 <widget class="QWidget" name="ToolsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tools</string>
  </property>
  <widget class="QToolButton" name="change_password">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>240</y>
     <width>221</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton {
	background: transparent;
	color: #3690ac;
}

QToolButton:hover {
	border: 1px solid #aaa;
	border-radius: 10px;
	background: #bed0e6;
}</string>
   </property>
   <property name="text">
    <string>Change Admin Password</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icon/change_password.png</normaloff>../icon/change_password.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>70</width>
     <height>70</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="administrationLabel">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>40</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: #e56f68;
}</string>
   </property>
   <property name="text">
    <string>Tools</string>
   </property>
  </widget>
  <widget class="QToolButton" name="settings">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>240</y>
     <width>161</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton {
	background: transparent;
	color: #3690ac;
}

QToolButton:hover {
	border: 1px solid #aaa;
	border-radius: 10px;
	background: #bed0e6;
}</string>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icon/setting.png</normaloff>../icon/setting.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="change_db">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>90</y>
     <width>161</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton {
	background: transparent;
	color: #3690ac;
}

QToolButton:hover {
	border: 1px solid #aaa;
	border-radius: 10px;
	background: #bed0e6;
}</string>
   </property>
   <property name="text">
    <string>Change Database</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icon/change_db.png</normaloff>../icon/change_db.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="backup_data">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>151</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton {
	background: transparent;
	color: #3690ac;
}

QToolButton:hover {
	border: 1px solid #aaa;
	border-radius: 10px;
	background: #bed0e6;
}</string>
   </property>
   <property name="text">
    <string>Backup Data</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icon/backup_db.png</normaloff>../icon/backup_db.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>70</width>
     <height>70</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="restore_data">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>90</y>
     <width>161</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton {
	background: transparent;
	color: #3690ac;
}

QToolButton:hover {
	border: 1px solid #aaa;
	border-radius: 10px;
	background: #bed0e6;
}</string>
   </property>
   <property name="text">
    <string>Restore Data</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icon/restore_db.png</normaloff>../icon/restore_db.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>70</width>
     <height>70</height>
    </size>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
